downstream/modules/platform/ref-system-requirements.adoc

```diff
@@ -12,13 +12,13 @@ Your system must meet the following minimum system requirements to install and r
 |===
 | Requirement | Required | Notes
 
-h| Subscription | Valid Red Hat Ansible Automation Platform |
+h| Subscription | Valid {PlatformName} |
 
 h| OS | Red Hat Enterprise Linux 8.6 or later 64-bit (x86, ppc64le, s390x, aarch64) |{PlatformName} is also supported on OpenShift, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/deploying_the_red_hat_ansible_automation_platform_operator_on_openshift_container_platform/index[Deploying the Red Hat Ansible Automation Platform operator on OpenShift Container Platform] for more information.
```

Comment:
I think we're using the attribute for RHEL now.

```suggestion
h| OS | {RHEL} 8.6 or later 64-bit (x86, ppc64le, s390x, aarch64) |{PlatformName} is also supported on OpenShift, see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/deploying_the_red_hat_ansible_automation_platform_operator_on_openshift_container_platform/index[Deploying the Red Hat Ansible Automation Platform operator on OpenShift Container Platform] for more information.
```

Reply:
Thanks. That slipped my mind.